docs/how-to/integrate_sdcore_with_observability.md

```diff
@@ -1,79 +1,105 @@
+deploy_cos = true
+cos_model_name = "YOUR_CUSTOM_COS_MODEL_NAME" (Optional. Defaults to `cos-lite`.)
+cos_configuration_config = {} (Optional. Allows customization of the `COS Configuration` application.)
 ```
 
-Integrate it with Grafana:
-
-```console
-juju integrate cos-configuration-k8s grafana
+```{note}
+If you have already created the `.tfvars` file, to customize the deployment of SD-Core, you should edit the existing file rather than create a new one.
 ```
 
-## Integrate Grafana Agent with Prometheus
+Proceed with the deployment.
+
+(option-2)=
+## Integrating COS with an existing SD-Core deployment
+
+This option allows deploying COS and integrating it with SD-Core as a Day 2 operation.
 
-We will create a cross model integration between Grafana Agent (in the SD-Core model) and Prometheus (in the `cos` model).
+### Pre-requisites
 
-First, offer the following integrations from Prometheus and Loki for use in other models:
+- A Kubernetes cluster capable of handling the load from both SD-Core and COS
+- Any [Charmed 5G Terraform module][Charmed 5G Terraform modules] deployed
 
-```bash
-juju offer cos.prometheus:receive-remote-write
-juju offer cos.loki:logging
+### Adding COS to an existing SD-Core deployment
+
+Go to a directory from which SD-Core was deployed (the one containing Terraform's `.tfstate` file).
+Edit the `.tfvars` and add following line(s) to it:
+
+```console
+deploy_cos = true
+cos_model_name = "<YOUR_CUSTOM_COS_MODEL_NAME>" (Optional. Defaults to `cos-lite`.)
+cos_configuration_config = {} (Optional. Allows customization of the `COS Configuration` application.)
 ```
 
-Then, consume the integrations from the SD-Core model:
+Apply the changes:
 
-```bash
-juju switch <SD-Core model>
-juju consume cos.prometheus
-juju consume cos.loki
+```console
+terraform apply -var-file="<YOUR_TFVARS_FILE>" -auto-approve
 ```
 
-Integrate `grafana-agent-k8s` with `prometheus` and `loki`:
+Monitor the status of the deployment:
 
-```bash
-juju integrate prometheus:receive-remote-write grafana-agent-k8s:send-remote-write
-juju integrate loki:logging grafana-agent-k8s:logging-consumer
+```console
+juju switch <YOUR_CUSTOM_COS_MODEL_NAME>
+watch -n 1 -c juju status --color --relations
 ```
 
+The deployment is ready when all the charms are in the `Active/Idle` state.
+
+## Accessing the 5G Network Overview Grafana dashboard
+
 Retrieve the Grafana URL and admin password:
 
 ```console
-ubuntu@host:~ $ juju run grafana/leader get-admin-password
+juju switch cos-lite
+juju run grafana/leader get-admin-password
+```
+
+You should see the output similar to the following:
+
+```console
 Running operation 1 with 1 task
   - task 2 on unit-grafana-0
 
 Waiting for task 2...
-admin-password: ngdrjomIOMyt
-url: http://10.0.0.5/cos-grafana
+admin-password: c72uEq8FyGRo
+url: http://10.201.0.51/cos-lite-grafana
+```
 
+```{note}
+Due to a bug in the Traefik charm, the URL returned by the command shown above, shows invalid `http` protocol.
```

Comment:
```suggestion
Due to a bug in the Traefik charm, the URL returned incorrectly shows `http` as the protocol.
```

Reply:
Good point. The old version suggested invalidity of the `http` as such.
I'd keep `by the command above,` tho. 
```Due to a bug in the Traefik charm, the URL returned by the command above, incorrectly shows `http` as the protocol.
```
WDYT?

Reply:
Other than the `,` in between the words above and incorrectly, looks good

Reply:
I've changed the note completely, because now both `http` and `https` work :)